src/main/resources/application.yaml

```diff
@@ -159,6 +159,7 @@ feature:
   secure-doc-store.enabled: ${SECURE_DOC_STORE_FEATURE:false}
   enhancedConfidentiality.enabled: ${ENHANCED_CONFIDENTIALITY_FEATURE:false}
   gaps-switchover.enabled: ${GAPS_SWITCHOVER_FEATURE:false}
+  work-allocation.enabled: ${WORK_ALLOCATION_FEATURE:false}
```

Comment:
Is this copied over from another PR / branch ?

Reply:
Nope I added this, can change if needed.

Reply:
Same as above. Please change to some case access management flag.

Reply:
Will be addressed in this PR: https://github.com/hmcts/sscs-common/pull/1178/files